File: src/game/entity/EntitySprites.java
package game.entity;

import game.graphics.ImageManager;
import game.graphics.Sprite;
import game.graphics.TextureAtlas;

/**
 * Created by thewa on 5/11/2016.
 */
public class EntitySprites {

    public static final TextureAtlas entitySpriteAtlas = new TextureAtlas(TextureAtlas.LARGE);

    public static final Sprite playerSprite = entitySpriteAtlas.addTexture("faded", ImageManager.getImage("/faded"));
}

File: src/game/entity/EntitySprites.kt
package game.entity

import game.graphics.ImageManager
import game.graphics.Sprite
import game.graphics.TextureAtlas

/**
 * Created by thewa on 5/11/2016.
 */
object EntitySprites {

    val entitySpriteAtlas = TextureAtlas(TextureAtlas.LARGE)

    val playerSprite = entitySpriteAtlas.addTexture("KnightFront", ImageManager.getImage("/entities/knight_front"))
}
